src/sentry/snuba/outcomes.py

```diff
@@ -294,10 +301,8 @@ def run_outcomes_query(query: QueryDefinition) -> Tuple[ResultSet, ResultSet]:
         referrer="outcomes.timeseries",
         limit=10000,
     )
-
-    result_totals = _format_rows(result["data"], query)
-    result_timeseries = _format_rows(result_timeseries["data"], query)
-    return result_totals, result_timeseries
+    formatted_results = _format_rows(result_timeseries["data"], query)
+    return formatted_results
```

Comment:
```suggestion
    return _format_rows(result_timeseries["data"], query)
```

You could fold these together 🏌️‍♂️